package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "windbreaker-server",
+  "version": "0.0.1",
+  "description": "",
+  "main": "src/server.js",
+  "scripts": {
+    "docker:build": "docker-compose build",
+    "docker:start": "docker-compose up rediscluster rabbitmq",
+    "docker:stop": "docker-compose down",
+    "lint": "eslint .",
+    "test": "npm run docker:build && docker-compose run test && docker-compose down",
+    "test:unit": "nyc ava test/unit",
+    "test:integration": "docker-compose run test npm run test:docker:integration",
+    "test:docker:ci": "npm run lint && nyc ava test/unit test/integration --verbose",
+    "test:docker:integration": "nyc ava test/unit test/integration --verbose",
+    "start": "browser-refresh src/server.js",
```

Comment:
It's actually a limitation of redis in NAT'd environments. Redis cluster will advertise IPs/ports which are routable from inside the docker network, but the host machine has no idea how to route to those addresses. The initial connect will work, but any requests which try to look up a key that is on a different master will result in the client getting redirected to an address which will be unroutable from the perspective of the host.